prac_06/programming_language.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 | Practical 06 - programming_language and languages  | Liam Eime
+Estimate: 20 minutes
+Actual:
+"""
+
+
+class ProgrammingLanguage:
+    """Represent programming language information"""
+
+    def __init__(self, name="", typing="", reflection=False, year=0):
```

Comment:
For your reflection object, are you sure you want the default value to only false? Maybe there is something better to set the value to.
Also, the same applies to the default value of the year object.